### ISSUE TITLE
Unable to login to Leetcode.com

### ISSUE THREAD

**Issue by BETTlM** [LeetCode-OpenSource/vscode-leetcode#946]
Whenever I click on sign in, I'm NOT able to see the option to choose cookies sign in at all, I don't even get that option. When I click on login, I get redirected to Leet code on my browser, and even after logging in from browser, the plugin gets struck on 'fetching user data' forever. Any fixes?

https://github.com/LeetCode-OpenSource/vscode-leetcode/assets/124069030/bbb45b7a-e734-4e3e-910c-799b5ce3ae39

**Comment by arroe99** [comment]
same problem

**Comment by TMMXA** [comment]
same problem

**Comment by WoDeXXZ** [comment]
I found a solution. You can download the previous version, log in and then update the plug-in.

**Comment by BETTlM** [comment]
> I found a solution. You can download the previous version, log in and then update the plug-in.

how do I download the previous version? Do i have to download from the github releases?

**Comment by BETTlM** [comment]
HELLO EVERYONE!! 
Thank you all for your concern, thank you @WoDeXXZ for your help. Here's how to fix this issue:

Install the .vsix file (the previous version of available in releases) of v0.17.0, and open your extensions tab in VS code and click on three dots to install the vsix file. After that you can sign in using the Cookie methods. Feel free to ping me regarding any difficulties. Once again, thank you so much @WoDeXXZ

**Comment by zhouzhongyuan** [comment]
https://marketplace.visualstudio.com/items?itemName=LeetCode.vscode-leetcode

Download from here

**Comment by BETTlM** [comment]
or, use this:
https://github.com/LeetCode-OpenSource/vscode-leetcode/releases/download/v0.16.2/shengchen.vscode-leetcode-0.16.2.vsix

**Comment by theashggl** [comment]
I don't think this works. I am still getting the same logs after using the previous version.
`Node.js v19.7.0
login: (node:15800) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)`

**Comment by BETTlM** [comment]
> I don't think this works. I am still getting the same logs after using the previous version. `Node.js v19.7.0 login: (node:15800) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency (Use `node --trace-warnings ...` to show where the warning was created)`

that was not my original issue, my issue was with the plugin mot nodejs. please refer the attachment i uploaded.

**Comment by theashggl** [comment]
> > I don't think this works. I am still getting the same logs after using the previous version. `Node.js v19.7.0 login: (node:15800) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency (Use `node --trace-warnings ...` to show where the warning was created)`
> 
> that was not my original issue, my issue was with the plugin mot nodejs. please refer the attachment i uploaded.

Yes, the issue is with the plugin while logging in. I also have the same issue as shown in the video. I just attached the logs from VSCode which is showing when that happens.

**Comment by BETTlM** [comment]
> > > I don't think this works. I am still getting the same logs after using the previous version. `Node.js v19.7.0 login: (node:15800) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency (Use `node --trace-warnings ...` to show where the warning was created)`
> > 
> > 
> > that was not my original issue, my issue was with the plugin mot nodejs. please refer the attachment i uploaded.
> 
> Yes, the issue is with the plugin while logging in. I also have the same issue as shown in the video. I just attached the logs from VSCode which is showing when that happens.

In that case, try regular sign-in with the latest version of the plugin.
I tried installing the latest version of the plugin and tried logging in using google pop-up, the pop-up sign in works just fine when I'm using Archlinux, but does not work when im using windows. I have no idea why this issue arises.

For windows, I just use the older version and it seems to work fine

**Comment by theashggl** [comment]
For me, on Windows, it doesn't work at all. I am not aware but have got the splash of it that it occurs due to node version so could you post your node version in windows for me to check?

> > > > I don't think this works. I am still getting the same logs after using the previous version. `Node.js v19.7.0 login: (node:15800) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency (Use `node --trace-warnings ...` to show where the warning was created)`
> > > 
> > > 
> > > that was not my original issue, my issue was with the plugin mot nodejs. please refer the attachment i uploaded.
> > 
> > 
> > Yes, the issue is with the plugin while logging in. I also have the same issue as shown in the video. I just attached the logs from VSCode which is showing when that happens.
> 
> In that case, try regular sign-in with the latest version of the plugin. I tried installing the latest version of the plugin and tried logging in using google pop-up, the pop-up sign in works just fine when I'm using Archlinux, but does not work when im using windows. I have no idea why this issue arises.
> 
> For windows, I just use the older version and it seems to work fine

**Comment by BETTlM** [comment]
> For me, on Windows, it doesn't work at all. I am not aware but have got the splash of it that it occurs due to node version so could you post your node version in windows for me to check?
> 
> > > > > I don't think this works. I am still getting the same logs after using the previous version. `Node.js v19.7.0 login: (node:15800) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency (Use `node --trace-warnings ...` to show where the warning was created)`
> > > > 
> > > > 
> > > > that was not my original issue, my issue was with the plugin mot nodejs. please refer the attachment i uploaded.
> > > 
> > > 
> > > Yes, the issue is with the plugin while logging in. I also have the same issue as shown in the video. I just attached the logs from VSCode which is showing when that happens.
> > 
> > 
> > In that case, try regular sign-in with the latest version of the plugin. I tried installing the latest version of the plugin and tried logging in using google pop-up, the pop-up sign in works just fine when I'm using Archlinux, but does not work when im using windows. I have no idea why this issue arises.
> > For windows, I just use the older version and it seems to work fine

I installed node on  windows and linux on the same day so this must be it

![image](https://github.com/LeetCode-OpenSource/vscode-leetcode/assets/124069030/a9877a13-1c75-4955-87c8-7f39e6cc8960)

**Comment by Abderaouf-KHELFAOUI** [comment]
@TMMXA I instaled the .vsix file and i still get the unfinite fetching user data!

**Comment by BETTlM** [comment]
> @TMMXA I instaled the .vsix file and i still get the unfinite fetching user data!

after gettin the vsix file, did you login with cookie or web?

**Comment by Abderaouf-KHELFAOUI** [comment]
it worked for me i've just to go for an older version of leetcode as you said.
thanks a lot!

**Comment by natesire** [comment]
I sent LeetCode's customer service an email regarding their broken login. LeetCode wants me to send diagnostics and screenshots of the login problem. I will make time for troubleshooting their website. Right now, I am dealing with HackerRank and erroneous practice problems with unit tests which contradict the word problems.
